apps/studio/components/interfaces/DiskManagement/ui/DiskManagementCoolDownSection.tsx

```diff
@@ -46,9 +44,10 @@ export const DiskMangementCoolDownSection: React.FC = () => {
           initial={{ opacity: 1, height: 'auto' }}
           exit={{ opacity: 0, height: 0 }}
           transition={{ duration: 0.15 }}
+          className="w-full"
         >
-          <DialogSection className="bg-surface-100 text-sm text-foreground-light flex items-center gap-4 relative">
-            <div className="w-12 h-12 opacity-75">
+          <DialogSection className="bg-surface-100 text-sm text-foreground-light flex items-center gap-4 relative w-full border rounded-md">
+            <div className="w-12 h-12">
```

Comment:
Modal makes me wonder how long restarts take, esp if my project is going to be down that whole time — maybe over thinking....

Reply:
No, you are right.

I have no way to calculate the downtime though.

If we can estimate we can for sure add it in.

Also it seems like it's not possible today to delay a restart, otherwise I would of added the option to Restart when convenient.

Reply:
👍  totally fine for a followup PR too.   
I think it's just expectation more than accurate estimate. 
Example: it's more important for me to know "I need to do this when most of my users are in bed" vs "This will take approximately 12 minutes". Just indicating downtime, which can't be reliably estimated should be fine